```
src/bin/sage -t --long --warn-long 30.0 --random-seed=286735480429121101562228604801325644303 src/sage/sets/recursively_enumerated_set.pyx
**********************************************************************
Error: Failed example:: Got: 
Traceback (most recent call last):
  File "/sage/src/sage/doctest/forker.py", line 724, in _run
    self.compile_and_execute(example, compiler, test.globs)
  File "/sage/src/sage/doctest/forker.py", line 1148, in compile_and_execute
    exec(compiled, globs)
  File "<doctest sage.sets.recursively_enumerated_set.RecursivelyEnumeratedSet_symmetric.graded_component_iterator[18]>", line 1, in <module>
    next(it)
  File "sage/sets/recursively_enumerated_set.pyx", line 1146, in graded_component_iterator
  File "sage/sets/recursively_enumerated_set.pyx", line 1237, in sage.sets.recursively_enumerated_set.RecursivelyEnumeratedSet_symmetric._get_next_graded_component
  File "<doctest sage.sets.recursively_enumerated_set.RecursivelyEnumeratedSet_symmetric.graded_component_iterator[13]>", line 5, in f
    raise KeyboardInterrupt
KeyboardInterrupt

    next(it)
Expected:
    Traceback (most recent call last):
    ...
    KeyboardInterrupt:
Got:
    <BLANKLINE>
    Traceback (most recent call last):
      File "/sage/src/sage/doctest/forker.py", line 724, in _run
        self.compile_and_execute(example, compiler, test.globs)
      File "/sage/src/sage/doctest/forker.py", line 1148, in compile_and_execute
        exec(compiled, globs)
      File "<doctest sage.sets.recursively_enumerated_set.RecursivelyEnumeratedSet_symmetric.graded_component_iterator[18]>", line 1, in <module>
        next(it)
      File "sage/sets/recursively_enumerated_set.pyx", line 1146, in graded_component_iterator
      File "sage/sets/recursively_enumerated_set.pyx", line 1237, in sage.sets.recursively_enumerated_set.RecursivelyEnumeratedSet_symmetric._get_next_graded_component
      File "<doctest sage.sets.recursively_enumerated_set.RecursivelyEnumeratedSet_symmetric.graded_component_iterator[13]>", line 5, in f
        raise KeyboardInterrupt
    KeyboardInterrupt
```

Why this is an error

--- a/src/sage/sets/recursively_enumerated_set.pyx
+++ b/src/sage/sets/recursively_enumerated_set.pyx
@@ -1132,7 +1132,7 @@ cdef class RecursivelyEnumeratedSet_symmetric(RecursivelyEnumeratedSet_generic):
             sage: next(it)
             Traceback (most recent call last):
             ...
-            KeyboardInterrupt:
+            KeyboardInterrupt
             sage: next(it)
             Traceback (most recent call last):
             ...

